recipe/build-r-base.sh

```diff
@@ -381,7 +381,7 @@ Mingw_w64_makefiles() {
 
     PREFIX_WIN=$(cygpath -w ${PREFIX})
 
-    sed -i "s|-lgfortran |$($FC --print-file libgfortran.dll.a) |g" "${PREFIX}"/lib/R/etc/${R_ARCH}/Makeconf
+    sed -i "s|-lgfortran |${PREFIX}/${HOST}/lib/libgfortran.dll.a |g" "${PREFIX}"/lib/R/etc/${R_ARCH}/Makeconf
```

Comment:
```suggestion
    sed -i "s|-lgfortran |${PREFIX}/Library/${HOST}/lib/libgfortran.dll.a |g" "${PREFIX}"/lib/R/etc/${R_ARCH}/Makeconf
```

Reply:
BTW, doesn't GCC normally have this in its standard search paths?
Is there something special in mingw so it's not the case?

Reply:
(Sorry, I'm rather noob-ish when it comes to mingw intricacies.)

Reply:
When only C compiler is in BUILD_PREFIX (no gfortran in BUILD_PREFIX) it does not know where to find `libgfortran.dll.a`. (This is an issue in windows only because on unix `$PREFIX/lib` is added by R.)

Reply:
No, I mean the search path for the static libs.
On the Unixes we dynamically link to `libgfortran.so` from `${PREFIX}/lib`.
But if you were to remove `${PREFIX}/lib/libgfortran.so` it could link to the static lib found in `${PREFIX}/${HOST}/lib` (cf. `gcc -print-search-dirs`).

So, question is: Is `${PREFIX}/Library/${HOST}/lib` not in library search path for MinGW or is it that it wouldn't look for a `libgfortran.dll.a` (rather than `libgfortran.a`) named one?

Reply:
Oh, nvm. I confused myself.

Reply:
> On the Unixes we dynamically link to libgfortran.so from ${PREFIX}/lib.
But if you were to remove ${PREFIX}/lib/libgfortran.so it could link to the static lib found in ${PREFIX}/${HOST}/lib (cf. gcc -print-search-dirs).

There's a subtle mistake here. It should have been

On the Unixes we dynamically link to libgfortran.so from ${PREFIX}/lib.
But if you were to remove ${PREFIX}/lib/libgfortran.so it could link to the static lib found in ${BUILD_PREFIX}/${HOST}/lib (cf. gcc -print-search-dirs).

Reply:
> There's a subtle mistake here.

true